portfolio/src/main/webapp/materialize.js

```diff
@@ -12412,24 +12412,51 @@ if (dark_mode == "no") {
 
 function createListElement(text) {
   const liElement = document.createElement('li');
-  liElement.innerText = text;
+  text = "    " + text;
+  liElement.innerHTML = "<button onclick=\"deleteComm()\" class=\"btn-small\"><i class=\"material-icons\">cancel</i></button>" + text;
   return liElement;
 }
+
+function createParagraphElem(text) {
+    const pElement = document.createElement('p');
+    pElement.innerText = text;
+    return pElement;
+}
+
+//function deleteComm(comment) {
+//    const params = new URLSearchParams();
+//    params.append('id', )
+//}
+
 var i;
 var curr_comment = 0;
+
+var num_comments;
+
 function getComments() {
     fetch('/data').then(response => response.json()).then((comments) => {
         const commentList = document.getElementById('comments');
         commentList.innerHTML = '';
-        for (i = 0; i < 5; i++) {
+        num_comments = 0;
+        i = 0;
+        for (i; i < 5; i++) {
             if (comments[i] != undefined) {
-                commentList.appendChild(createListElement(comments[i]))
+                commentList.appendChild(createListElement(comments[i]));
             }
             curr_comment++;
         }
+        while (comments[num_comments] != undefined) {
+            num_comments++;
+        }
+        var descriptor = "Page " + Math.ceil(i/5) + " of " + Math.ceil(num_comments/5);
```

Comment:
This and below is repeated three times, maybe worthy to give a function ?

Reply:
Done.